lib/topological_inventory/core/version.rb

```diff
@@ -1,5 +1,5 @@
 module TopologicalInventory
   module Core
-    VERSION = '1.0.0'
```

Comment:
@syncrou this is "interesting" - I thought PRs are compared to the master, but master contains version 1.1.4.
https://github.com/RedHatInsights/topological_inventory-core/blob/master/lib/topological_inventory/core/version.rb
So it seems it'll be much harder to follow it for more parallel PRs

@pkomanek please rebase your PR, you should update to 1.1.5

Reply:
@slemrmartin I'm guessing @pkomanek had a very old version of master?  It seems okay now when I look at the diff.